### PR TITLE
removed all visibles

### PR DESCRIPTION
**Pull request by smptr0911**
I removed all the visibles

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -9,25 +9,9 @@ function App() {
...
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
+      <div className=" ">
       </div>
```
Comment (dbigueta) [inline src/App.jsx]: If these are empty, you should just delete it

**Review by dbigueta — CHANGES_REQUESTED** [review]
If you're not using the imports, you should also delete them, we don't want unused code going into the main branch

File: src/App.jsx (hunk 1 above)
Comment (smptr0911) [reply src/App.jsx]: Done!

**Review by dbigueta — APPROVED** [review]
Perfect!

**Pull request merged**